docs/developer/explanations/decisions/0003-added-make-devices-decorator.rst

```diff
@@ -0,0 +1,33 @@
+3. Adopt python3-pip-skeleton for project structure
```

Comment:
Wrong title

On that note: Could this PR also have a better title?

Reply:
yes, it does. I keep forgetting that for some reason github takes the first commit as the PR name instead of the branch name. I usually make the first commit 'first look exploration' and want to make the PR early. not sure yet what to do with this, will think about it

Reply:
I like early PRs, so maybe you just have to keep more of an eye on it?

Reply:
I could write the first commit as usual, then just change the name just after the PR is made